src/Apps/Legion/Hooks/useEntities.tsx

```diff
@@ -124,9 +129,18 @@ export const useEntities = () => {
              */
             deleteEntity: deleteEntity,
             /** Callback to get entity count per state based on Type */
-            getEntityCount
+            getEntityCount: getEntityCount,
```

Comment:
can we not use the existing count function? I think it returns an object and one of those values in teh "new" items?

Reply:
Oops I didn't realize it was looking at the same property! Swapping